dashbuilder-client/dashbuilder-renderers/dashbuilder-renderer-default/src/test/java/org/dashbuilder/renderer/client/selector/SelectorSliderDisplayerTest.java

```diff
@@ -0,0 +1,225 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.dashbuilder.renderer.client.selector;
+
+import java.util.Date;
+
+import org.dashbuilder.dataset.client.engine.ClientDateFormatter;
+import org.dashbuilder.dataset.filter.CoreFunctionFilter;
+import org.dashbuilder.dataset.filter.CoreFunctionType;
+import org.dashbuilder.dataset.filter.DataSetFilter;
+import org.dashbuilder.dataset.filter.FilterFactory;
+import org.dashbuilder.displayer.DisplayerSettings;
+import org.dashbuilder.displayer.DisplayerSettingsFactory;
+import org.dashbuilder.displayer.client.AbstractDisplayerTest;
+import org.dashbuilder.displayer.client.DisplayerListener;
+import org.dashbuilder.displayer.client.widgets.filter.DateParameterEditor;
+import org.dashbuilder.displayer.client.widgets.filter.NumberParameterEditor;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import static org.dashbuilder.dataset.ExpenseReportsData.*;
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
+
+@RunWith(MockitoJUnitRunner.class)
+public class SelectorSliderDisplayerTest extends AbstractDisplayerTest {
```

Comment:
One more issue with a slider that I identified:
When upper and lower slider values are equal, it seems to be filtering data from some larger range as in the following screenshot: I guess that should be reported as well ?
![screenshot from 2017-03-08 10-07-41](https://cloud.githubusercontent.com/assets/2716069/23697219/3a7f8844-03e7-11e7-9e02-f06f34afc413.png)